rider/frontend.gradle

```diff
@@ -65,7 +65,7 @@ intellij {
     downloadSources = false
     instrumentCode = false
 
-    plugins = [ 'rider-plugins-appender', 'CSS', 'yaml' ]
+    plugins = [ 'rider-plugins-appender', 'CSS', 'yaml', 'dotCover' ]
```

Comment:
what would happen if user doesn't have dotCover?

Reply:
java.lang.NoClassDefFoundError: com/jetbrains/dotCover/RiderDotCoverService
	at com.jetbrains.rider.test.scriptingApi.RiderDotCoverScriptingFacade.<init>(ScriptingAPI.DotCover.kt:43)
	at com.jetbrains.rider.test.scriptingApi.ScriptingAPI_DotCoverKt$withDcFacade$1.invoke(ScriptingAPI.DotCover.kt:294)
	at com.jetbrains.rider.test.scriptingApi.ScriptingAPI_DotCoverKt$withDcFacade$1.invoke(ScriptingAPI.DotCover.kt)
	at com.jetbrains.rider.test.scriptingApi.ScriptingAPI_UnitTestingKt.withUtFacade(ScriptingAPI.UnitTesting.kt:797)
	at com.jetbrains.rider.test.scriptingApi.ScriptingAPI_UnitTestingKt.withUtFacade$default(ScriptingAPI.UnitTesting.kt:790)
	at com.jetbrains.rider.test.scriptingApi.ScriptingAPI_DotCoverKt.withDcFacade(ScriptingAPI.DotCover.kt:293)
	at integrationTests.DotCoverTest.checkCoverAllTestsFromSolution(DotCoverTest.kt:26)

Reply:
I mean wouldn't it brake the plugin for users, who don't have dotCover plugin installed?

Reply:
It won't break, because we don't add a dependency to dotCover in plugin.xml using tag <depends> (we can't do this, because dotCover has dependency on unity plugin), in this place we just bundle dotCover in SDK